include/triton/Tools/Sys/GetEnv.hpp

```diff
@@ -55,6 +56,18 @@ inline std::string getenv(const char *name) {
   return result;
 }
 
+inline std::vector<std::string> getenv(const char *name, char split) {
+  std::string str = triton::tools::getenv(name);
+  if (str.empty())
+    return {};
+  std::vector<std::string> tokens;
+  std::stringstream sstr(str);
+  std::string token;
+  while (std::getline(sstr, token, ','))
```

Comment:
Should we use split here? The rest looks good to me. Thanks for making the change.

Reply:
I see, nice catch!